buttons.css

```diff
@@ -42,4 +42,8 @@
 
 .btn-icon svg {
     height: 1em;
+}
+
+.btn-transparent {
+    color: var(--navy);
```

Comment:
This is called transparent but changes the color to `navy`. Something is fishy here. Maybe rename?